Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM elixir:1.4.2
+
+ENV NODE_VERSION=7
+
+# Install system dependencies
+RUN apt-get -y update \
+    && apt-get -y install apt-transport-https build-essential curl git make locales locales-all inotify-tools
+
+# Install nodejs
+RUN curl -sL https://deb.nodesource.com/setup_$NODE_VERSION.x | bash - \
+    && apt-get -y install nodejs
+
+# Clean up apt temporary apt artefacts
+RUN apt-get clean -y \
+    && rm -rf /var/cache/apt/*
```

Comment:
what is the intention of this command?

It will be a separate layer so the previous layers will still have the cache hence you don't win on the size of the image. For this to be useful you need to chain all the previous apt commands into one RUN.